aws/resource_aws_dms_endpoint.go

```diff
@@ -117,6 +118,39 @@ func resourceAwsDmsEndpoint() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"mongodb_settings": {
```

Comment:
This attribute is missing documentation. 😄 

We may start seeing differences that are something like this if the argument is not present in the Terraform configuration (regardless of whether or not all the children attributes are the default values):

```
mongodb_settings.#: "1" => "0"
```

Due to some limitations with Terraform's helper/schema, we will likely need to workaround this with a [DiffSuppressFunc](https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#diffsuppressfunc) on the `mongodb_settings` attribute:

```go
DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
  if old == "1" && new == "0" {
    return true
  }
  return false
},
```